src/opera/pge/disp_s1/disp_s1_pge.py

```diff
@@ -27,7 +28,10 @@
 from opera.util.h5_utils import get_disp_s1_product_metadata
 from opera.util.input_validation import validate_algorithm_parameters_config, validate_disp_inputs
 from opera.util.render_jinja2 import render_jinja2
-from opera.util.time import get_time_for_filename
+from opera.util.time import get_time_for_filename, get_catalog_metadata_datetime_str
+
+
+MEASURED_PARAMETER_PATH_SEPARATOR = '/'
```

Comment:
Is this constant defined in multiple places at this point? Maybe we should refactor to a single location in `h5_util.py`, since it's only needed when defining the measured parameter config for PGE's that produce HDF5 outputs.

Reply:
Done